bots/cli/src/main/java/org/openjdk/skara/bots/cli/BotLogstashHandler.java

```diff
@@ -78,7 +78,7 @@ private void publishToLogstash(Instant time, Level level, String message, Map<St
             query.put("level_value", level.intValue());
             query.put("message", message);
 
-            for (Map.Entry<String, String> entry : LogContextMap.entrySet()) {
+            for (var entry : LogContextMap.entrySet()) {
```

Comment:
Right, not used to var yet. I use Intellij's templates to generate loops and the template generates the type for me. I did change it manually in some other location but missed this one.